conda-store-server/conda_store_server/server/views/ui.py

```diff
@@ -63,6 +64,7 @@ async def ui_list_environments(
             "environments": orm_environments.all(),
             "registry_external_url": server.registry_external_url,
             "entity": entity,
+            "platform": "Windows" if sys.platform == "win32" else "Linux, macOS",
```

Comment:
This needs to be updated because Linux and macOS installers are not compatible: cannot use a Linux installer on macOS